infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/GeometryService.kt

```diff
@@ -343,6 +356,36 @@ class GeometryService @Autowired constructor(
         return FileName("$VERTICAL_GEOMETRY ${locationTrack.name}") to csvFileContent.toByteArray()
     }
 
+    @Scheduled(
+        cron = "\${geoviite.rail-network-export.schedule}"
```

Comment:
Jos/kun tämä käyttää samaa schedulea kuin elementtilistauksen genis, niin nämä taitaa ajautua aina suurin piirtein yhtä aikaa ja ensimmäiseksi ajautumaan lähtevä taitaa aina blokata myöhemmän. Eli näille voisi varmaankin joko eri schedulet, tai sitten nämä molemmat voisi ajaa samasta entrypointista saman `runWithLock`:in sisällä

Reply:
Jeppis. Lisäsin uuden ajoituksen tätä varten.